mods/common/languages/rules/en.ftl

```diff
@@ -0,0 +1,58 @@
+## Shroud
+checkbox-fog-of-war =
+    .label = Fog of War
+    .description = Line of sight is required to view enemy forces
+
+checkbox-explored-map =
+    .label = Explored Map
+    .description = Initial map shroud is revealed
+
+## DeveloperMode
+checkbox-debug-menu =
+    .label = Debug Menu
+    .description = Enables cheats and developer commands
+
+## CrateSpawner
+checkbox-crates =
+    .label = Crates
+    .description = Collect crates with units to receive random bonuses or penalties
+
+## MapBuildRadius
+checkbox-ally-build-radius =
+    .label = Build off Allies
+    .description = Allow allies to place structures inside your build area
+
+checkbox-build-radius =
+    .label = Limit Build Area
+    .description = Limits structure placement to areas around Construction Yards
+
+## MapOptions
+checkbox-short-game =
+    .label = Short Game
+    .description = Players are defeated when their bases are destroyed
+
+dropdown-tech-level =
+    .label = Tech Level
+    .description = The units and abilities that players can use
+
+## MapOptions
+dropdown-game-speed =
+    .label = Game Speed
+    .description = The rate at which time passes
+
+## MapStartingLocations
+checkbox-separate-team-spawns =
+    .label = Separate Team Spawns
+    .description = Players without assigned spawn points will start as far as possible from enemy players
+
+## SpawnStartingUnits
+dropdown-starting-units =
+    .label = Starting Units
+    .description = The units that players start the game with
+
+## TimeLimitManager
+dropdown-time-limit =
+    .label = Time Limit
+    .description = Player or team with the highest score after this time wins
+
+notification-time-limit-expired = Time limit has expired.
```

Comment:
This is duplicated now. (See the last line of the `en.ftl` this was moved from.) Is the lint rule not working?

Reply:
Ah I see the lint test mentioned in the OP just tests if a map duplicates a key.

Reply:
removed duplicate